### DISCUSSION TITLE
ESP32 DEVKIT V1 tasmo compiler

### DISCUSSION THREAD

**Discussion by mmmmm009** [arendst/Tasmota#20843] [Q&A] [answered]
Hi,
I have this board:
![ESP32-DOIT-DEV-KIT-v1-pinout-mischianti (1)](https://github.com/arendst/Tasmota/assets/72743330/c59c484b-739e-4734-b257-9e65595a4b20)

Im using Tasmo Compiler to enable SCRIPT.
![tasmo setting](https://github.com/arendst/Tasmota/assets/72743330/14823088-f42c-46dc-9848-60f2b9193b8f)

I've tried all possibilities: 
ESP32 generic, solo1, C3, S2, S3 and it doesn't work.
I keep getting this:

`rst:0x10 (RTCWDT_RTC_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
invalid header: 0x44203733
invalid header: 0x44203733
invalid header: 0x44203733
invalid header: 0x44203733
invalid header: 0x44203733
invalid header: 0x44203733
invalid header: 0x44203733
invalid header: 0x44203733
ets Jul 29 2019 12:21:46
`

Any idea on how to flash ESP32 DEVKIT V1 using tasmo compiler?
THX!

**Comment by barbudor** [comment, marked as answer]
For initial serial flashing use the factory.bin file
.bin is only for upgrade

You can also install a standard tasmota using the web installer and then upgrade to your .bin (not factory this time)

**Reply by sfromis** [reply]
And do not select ESP32-S3, that one is "generic".
